src/audio/generic_processor/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-add_local_sources(sof generic_processor.c)
+add_local_sources(sof generic_processor.c hifi_codec/codec.c)
```

Comment:
hifi_codec name suggests specialization for Hifi co-processor while this layer supports all Cadence Codec API modules and there is no direct reference to any Hifi code.